static/js/controllers/configuration-forms.js

```diff
@@ -89,10 +89,10 @@ angular.module('inboxControllers').controller('ConfigurationFormsCtrl',
           }
 
           var update = function(doc) {
-            doc.context = context;
             doc.title = title;
             doc.type = 'form';
             doc.internalId = formId;
+            _.extend(doc, meta);
```

Comment:
It's debatable whether this file should overwrite `.internalId` or `.title`.  It definitely shouldn't be overriding `.type`.